shapely/ops.py

```diff
@@ -211,11 +211,14 @@ def id_func(x, y, z=None):
 
       project = partial(
           pyproj.transform,
-          pyproj.Proj(init='epsg:4326'),
-          pyproj.Proj(init='epsg:26913'))
+          pyproj.Proj('epsg:4326', always_xy=True),
+          pyproj.Proj('epsg:26913', always_xy=True))
```

Comment:
I would personally update it here to use CRS as well (which I think is almost always to prefer above Proj, nowadays)